### ISSUE TITLE
pi-zmq zeromq integration as a publisher

### ISSUE THREAD

**Issue by bc** [bc/pi-zmq#10]
https://www.linkedin.com/pulse/tests-ensuring-control-system-sampling-rate-brian-cohn/

example:
https://media.licdn.com/dms/image/C5612AQH-SQUP5g4dkA/article-inline_image-shrink_1000_1488/0?e=1538611200&v=beta&t=xbOlCiUcoGtbNcnBXCijKGCQxy9NZuAy_cOV62EyJaI

1. Run a babble via your laptop on `python3 pyproctor/main.py` with 10000 babble samples to get a baseline.
2. Write a small python3 function that takes in a response CSV (the file generated from number 1), and outputs
a. mean and standard deviation of the inter-sample intervals. function to convert time_0 col into inter sample intervals: https://docs.scipy.org/doc/numpy-1.14.0/reference/generated/numpy.diff.html
3. convert to sampling frequency, and record the mean and SD, it should be close to 80SPS
4. add a zeromq publisher within the control loop to stream the observation out: this is the list you'd be sending out:
https://github.com/bc/pi-zmq/blob/iss9/t2.py#L88
5. Get the subscriber timeseries visualization working with your laptop (showing the signals of m0,m1,m2,e0,e1 as streams, over time_0)
6. re-run babble while it's streaming
7. re-run babble while it's not streaming
8. re-run babble while it's streaming, then unplug halfway through
9. evaluate sampling frequency, interval for each. evaluate whether we need to change the sleep time to accommodate for the ZMQ activity (sleep line: https://github.com/bc/pi-zmq/blob/iss9/t2.py#L91)

TLDR: implement zmq and test whether it negatively impacts sampling frequency, inter-sample-interval variability, or both.

why: then we can have pretty live plots
